Makefile.opensuse

```diff
@@ -1,19 +1,9 @@
 include Makefile
 
+OS_NAME:=opensuse
+REPO_BASE:=registry.opensuse.org/opensuse/
 SERVER_SRC_FILE:=$(SERVER_DIR)/Containerfile.opensuse
 AD_SERVER_SRC_FILE:=$(AD_SERVER_DIR)/Containerfile.opensuse
 CLIENT_SRC_FILE:=$(CLIENT_DIR)/Containerfile.opensuse
 TOOLBOX_SRC_FILE:=$(TOOLBOX_DIR)/Containerfile.opensuse
 SERVER_SOURCES:=$(SERVER_DIR)/smb.conf
-SERVER_NAME:=samba-opensuse-container:$(TAG)
-AD_SERVER_NAME:=samba-ad-opensuse-container:$(TAG)
-CLIENT_NAME:=samba-client-opensuse-container:$(TAG)
-TOOLBOX_NAME:=samba-toolbox-opensuse-container:$(TAG)
-SERVER_REPO_NAME:=registry.opensuse.org/opensuse/samba-server:$(TAG)
-AD_SERVER_REPO_NAME:=registry.opensuse.org/opensuse/samba-ad-server:$(TAG)
-CLIENT_REPO_NAME:=registry.opensuse.org/opensuse/samba-client:$(TAG)
-TOOLBOX_REPO_NAME:=registry.opensuse.org/opensuse/samba-toolbox:$(TAG)
-BUILDFILE_SERVER:=.build.opensuse-server
-BUILDFILE_AD_SERVER:=.build.opensuse-ad-server
-BUILDFILE_CLIENT:=.build.opensuse-client
-BUILDFILE_TOOLBOX:=.build.opensuse-toolbox
```

Comment:
These lines change the name of the repo at opensuse registry. I don't think this is something we can do without cooperation from dmulder nor is it something I want to do.

Reply:
> These lines change the name of the repo at opensuse registry. I don't think this is something we can do without cooperation from dmulder nor is it something I want to do.

I agree, I meant to change this before pushing, but I seem to have forgotten,,, workin on fixing it now

Reply:
These repo names are actually set by the `org.opensuse.reference` label in Containerfile.opensuse in our build system. The only time we use these `*_REPO_NAME` labels is for local builds. So changing them wont break our builds in any way.

This brings up an interesting problem... Consider what happens when somebody does a `make -f Makefile.opensuse build-server` followed by a `make push-server` (without the opensuse Makefile). Wont this push an opensuse image to quay.io, overwriting the fedora image?

Reply:
> These repo names are actually set by the `org.opensuse.reference` label in Containerfile.opensuse in our build system. The only time we use these `*_REPO_NAME` labels is for local builds. So changing them wont break our builds in any way.
> 
That's helpful, thanks!

> This brings up an interesting problem... Consider what happens when somebody does a `make -f Makefile.opensuse build-server` followed by a `make push-server` (without the opensuse Makefile). Wont this push an opensuse image to quay.io, overwriting the fedora image?

I've been working with @obnoxxx on this PR and what I'm aiming for is to have the "distro variants", the `Makefile -f Makefile.<distro>` makefiles produce tags that include the distro name, like "fedora-latest", "fedora-nightly", "centos-latest", "opensuse-latest".  The default makefile will continue to produce tags "latest" and "nightly".  These tags apply to the "short name" and the "repo name".

So I think we'd be OK since running `make -f Makefile.opensuse build-server && make push-server` would build images named `samba-server:opensuse-latest` and `registry.opensuse.org/opensuse/samba-server:latest` [1] but the push will attempt to push `quay.io/samba.org/samba-server:latest` (which may or may not exist in the local cache, but shouldn't conflct with the opensuse build).

At least that's the intent :-)

[1] - Based on  suggestions @obnoxxx  and I  discussed yesterday on a 1:1 call.  Those changes are not pushed yet AFAICT